share/clus_apis.go

```diff
@@ -2712,3 +2723,28 @@ type CLUSApikey struct {
 	CreatedTimestamp    int64               `json:"created_timestamp"`
 	CreatedByEntity     string              `json:"created_by_entity"` // it could be username or apikey (access key)
 }
+
```

Comment:
1. Please add "Comment" & "CfgType" fields
2. Should we add a field "PrivateRoot bool json:"private_root"" ?